ts/packages/node/src/stores/TomlFile.ts

```diff
@@ -0,0 +1,80 @@
+import { ConfigStore, ConfigStoreQuery, Config, Convert } from '@configu/ts';
+import _ from 'lodash';
+import { promises as fs } from 'fs';
+import toml from '@iarna/toml';
+
+export type TomlFileConfigStoreConfiguration = { path: string };
+
+export class TomlFileConfigStore extends ConfigStore {
```

Comment:
Please change `TomlFileConfigStore` to extend `FileConfigStore`. You can take reference from the INI config store regarding the implementation due to the similarities between INI and TOML files.